include/grpc++/fixed_size_thread_pool.h

```diff
@@ -45,10 +45,10 @@
 
 namespace grpc {
 
-class ThreadPool GRPC_FINAL : public ThreadPoolInterface {
+class FixedSizeThreadPool GRPC_FINAL : public ThreadPoolInterface {
  public:
-  explicit ThreadPool(int num_threads);
-  ~ThreadPool();
+  explicit FixedSizeThreadPool(int num_threads);
+  ~FixedSizeThreadPool();
 
   void ScheduleCallback(const std::function<void()>& callback) GRPC_OVERRIDE;
```

Comment:
I'm good with it

On Thu, Jul 16, 2015, 6:25 PM Abhishek Kumar <notifications@github.com>
wrote:

> In include/grpc++/fixed_size_thread_pool.h
> <https://github.com/grpc/grpc/pull/2460#discussion_r34855565>:
>
> >   public:
> > -  explicit ThreadPool(int num_threads);
> > -  ~ThreadPool();
> > +  explicit FixedSizeThreadPool(int num_threads);
> > +  ~FixedSizeThreadPool();
> >
> >    void ScheduleCallback(const std::function<void()>& callback) GRPC_OVERRIDE;
>
> What do you think of renaming ScheduleCallback() to Add(), to be
> consistent with other threadpools at google?
> I know this is not touched in this PR, but now that we are making this
> into a public API, it might be a good time to clean up names ..
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grpc/grpc/pull/2460/files#r34855565>.
>